lib/action_subscriber.rb

```diff
@@ -80,6 +80,10 @@ def self.start_subscribers
     print_subscriptions
   end
 
+  def self.stop_subscribers!
+    @route_set.cancel_consumers! if @route_set
```

Comment:
this code path is called during a graceful shutdown so I'm just guarding against the race condition of someone starting up the subscriber and killing it before ActionSubscriber has actually finished starting up. I didn't want to throw a false warning, but maybe I should add guard to the other places we access the route set?